llvm/test/CodeGen/AArch64/GlobalISel/combine_urem.ll

```diff
@@ -0,0 +1,243 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple=aarch64-unknown-unknown | FileCheck %s --check-prefixes=CHECK-SD
+; RUN: llc < %s -mtriple=aarch64-unknown-unknown -global-isel | FileCheck %s --check-prefixes=CHECK-GI
```

Comment:
If you're going to mix dag and globalisel run lines, should explicitly disable global-isel in the dag line.  Also is there an existing test you can just add run lines to?